README.rst

```diff
@@ -70,14 +70,11 @@ To enable logging to a separate file use the following configuration:
     zope-conf-additional +=
         <product-config collective.fingerpointing>
             audit-log ${buildout:directory}/var/log/audit.log
-            audit-log-max-size 10240
```

Comment:
thanks for your clarification @hvelarde. having this feature in place i worry much less about rotating logs on a daily basis by default.
(i did not try out new releases/features since the last time i commited to this repository - so i'm not completely up to date)